app/renderer/src/components/Icon.js

```diff
@@ -24,9 +24,19 @@ const iconImgText = compose(
 );
 
 const Icon = (props) => {
+
+  const iconTextStyle = props.icon.bgColor
+    ? compose(
+      iconImgTextBase,
+      props.icon.bgColor && style({
```

Comment:
is the `props.icon.bgColor` needed?
it should have passed the ternary condition at this point